src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/integration/listener/ExtractPlacedEventListenerTest.kt

```diff
@@ -28,6 +30,8 @@ class ExtractPlacedEventListenerTestTest : IntegrationTestBase() {
     reductionReason = reductionReasonRepository.save(ReductionReasonEntity(reductionCategoryEntity = reductionCategory))
 
     wmtStaff = setupCurrentWmtStaff("STAFF2", "TEAM2")
+    staffCodeResponse(wmtStaff.offenderManager.code, wmtStaff.team.code)
+    staffCodeResponse(wmtStaff.offenderManager.code, wmtStaff.team.code)
```

Comment:
Intended duplicate?

Reply:
it was, yeah, I've removed a call to community API now so now only 1 call is needed